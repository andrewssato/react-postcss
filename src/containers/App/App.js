import React, { Component } from 'react';
import style from './style.css';

class App extends Component {
  render() {
    return (
      <div className={style.app}>
          Test Postcss app
      </div>
    );
  }
}

export default App;
