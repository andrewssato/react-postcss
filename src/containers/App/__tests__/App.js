import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from './..';

describe('#App', () => {
  // to remove warnings
  console.error = () => null;

  // mock style postcss
  const theme = {app: 'app'};

  it('should render App component', () => {
    const el = shallow(<App theme={theme} />);
    expect(el.find('div')).toHaveLength(1);
  });

  it('Generate source of App without properties', () => {
    const tree = renderer.create(<App theme={theme} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
