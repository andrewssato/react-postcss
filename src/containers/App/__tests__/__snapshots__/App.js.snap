// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`source de app sem propriedades 1`] = `
<div
  className={undefined}
>
  Test Postcss app
</div>
`;
